client/services/SolutionHandlerService.js

```diff
@@ -63,6 +63,7 @@ tie.factory('SolutionHandlerService', [
             CodePreprocessorDispatcherService.preprocess(
               language, codeSubmission, auxiliaryCode, tasks);
 
+
```

Comment:
Remove extra line here.

In general, it's a good idea to check the diff immediately after submitting (or immediately before using `git diff` or `meld`, and then you can fix it as needed prior to review. It's the "Files Changed" tab -- see https://github.com/google/tie/pull/320/files/a39e989ad9d6c869fe834ea0e0c8804a2c1946c8..ed03e64d36dfb1fe68f15cfcf6332be200c096fd